FlickrApp/View/Tab/SearchTab.swift

```diff
@@ -0,0 +1,88 @@
+//  FlickrApp
+//
+//  Created by Suyash Srivastav on 20/02/24.
+//
+
+import Foundation
+import SwiftUI
+
+struct SearchTab: View {
+    @Binding var searchText: String
+    @ObservedObject var viewModel: PhotoViewModel
+    @State private var isLoading = false
+
+    var body: some View {
+        VStack(spacing: 0) {
+            Text("Flickr Search")
+                .font(.system(size: 24))
+            SearchBar(text: $searchText, onSearch: {
+                isLoading = true
+                viewModel.searchPhotos(query: searchText)
+            })
+            .padding()
+            ScrollView {
+                LazyVGrid(columns: [GridItem(.flexible()), GridItem(.flexible())], spacing: 10) {
```

Comment:
Let's try to divide this in multiple views ? maybe dividing this views in different computed property/function will be cleaner.